resources/js/components/Common/Banner.vue

```diff
@@ -1,17 +1,15 @@
 <script setup lang="ts">
 import { XMarkIcon } from '@heroicons/vue/20/solid'
-defineProps<{text:string}>()
+defineProps<{text:string, bannerClass?:string}>()
```

Comment:
You don't need define `class` prop for first, main element in template. You can just write `class` where you use this component.